infra/gcp/modules/gke-cluster/outputs.tf

```diff
@@ -13,6 +13,10 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+output "cluster" {
+  description = "The cluster"
+  value       = google_container_cluster.cluster
+}
```

Comment:
You might consider outputting ca_certificate, and endpoint for the GKE cluster so you can setup helm or kubectl from in TF later on.

Reply:
I _think_ they can access from `outputs.cluster.xyz`? Not sure though